contributor-dashboard-legacy/src/pages/Landing.js

```diff
@@ -347,14 +193,15 @@ function useRecentBuilds(filter) {
 	}, [branchName, builds, workflowName]);
 
 	// Fetch as long as we didn't find a build but stop after X pages.
-	if (filteredBuilds.length === 0 && page < 10) {
+	if (filteredBuilds.length === 0 && page < 9) {
 		setPage(page + 1);
 	}
 
-	return useMemo(() => filteredBuilds.slice(0, 20), [filteredBuilds]);
+	return React.useMemo(() => filteredBuilds.slice(0, 20), [filteredBuilds]);
 }
 
 async function fetchRecentCircleCIBuilds(key, cursor = 0) {
+	console.log("cursor", cursor);
```

Comment:
stray console log?

Reply:
This logic is broken, it fetch page 0 and 9 and ignores the ones in between 😂, but we don't really care, we should likely delete all of that Landing page. Applying the proposed change.